core/src/main/java/com/google/zxing/qrcode/decoder/DecodedBitStreamParser.java

```diff
@@ -160,7 +160,7 @@ private static void decodeHanziSegment(BitSource bits,
       // Each 13 bits encodes a 2-byte character
       int twoBytes = bits.readBits(13);
       int assembledTwoBytes = ((twoBytes / 0x060) << 8) | (twoBytes % 0x060);
-      if (assembledTwoBytes < 0x003BF) {
+      if (assembledTwoBytes < 0x00A00) {
```

Comment:
0x0A00 + 0xA1A1 is beyond 0xAAFE though; just checking this is really the right bound?

Reply:
There is no character mapped between 0xAAFF and 0xB0A0 in GB2312. So you can choose one from 0x95E to 0xA00.